experiments/pid/config_overrides/quadrotor_stabilization.yaml

```diff
@@ -43,3 +43,9 @@ task_config:
   episode_len_sec: 6
   cost: quadratic
   done_on_out_of_bound: False
+
+algo_config:
```

Comment:
could be cleaner to simply have a pid_config.yaml file with this instead of adding it in both quadrotor yamls

Reply:
oh yes, sorry. this is quick hack to pass the test. i will make a separate ctrl config and modify it in the pid_experiment.sh